pandas/io/formats/style.py

```diff
@@ -2025,3 +2047,28 @@ def _maybe_wrap_formatter(
     else:
         msg = f"Expected a string, got {na_rep} instead"
         raise TypeError(msg)
+
+
+def _maybe_convert_css_to_tuples(
+    style: Union[str, Sequence[Tuple[str, Union[str, int, float]]]]
+):
```

Comment:
can you add an output type (maybe define this at the top as CSSProperties or similar)